src/processing/farger/farger.md

```diff
@@ -6,92 +6,98 @@ author: Sigmund Hansen
+  ```processing
+  void setup() {
+    // bestem størrelse til vinduet
+    size(800, 600);
+  }
+
+  void draw() {
+    background(0);
+  }
+  ```
 
   Dette har du kanskje sett før. Når vi kaller på `background` med
   bare ett tall får vi en gråtone der `0` er sort og `255` er hvitt.
 
 + La oss endre på `draw` slik at vi får en rød bakgrunn:
 
-    ```processing
-    void draw() {
-      background(255, 0, 0);
-    }
-    ```
+  ```processing
+  void draw() {
+    background(255, 0, 0);
+  }
+  ```
 
   Dette likner på det vi hadde i det første steget, men nå bruker vi
-  plutselig tre tall istedenfor ett. La oss utforske dem litt.
+  plutselig tre tall istedenfor ett. La oss utforske dem.
 
-+ La oss endre på kallet på `background` igjen:
++ La oss endre `background` igjen:
 
-    ```processing
-    void draw() {
-      background(0, 255, 0);
-    }
-    ```
+  ```processing
+  void draw() {
+    background(0, 255, 0);
+  }
+  ```
 
-  Hvis du kjører programmet, får du nå en grønn bakgrunn istedenfor en
-  rød.
+  Når du kjører programmet, hvilken farge får du på bakgrunnen?
 
-+ La oss endre på kallet på `background` enda en gang:
++ La oss endre `background` enda en gang:
 
-    ```processing
-    void draw() {
-      background(0, 0, 255);
-    }
-    ```
+  ```processing
+  void draw() {
+    background(0, 0, 255);
+  }
+  ```
 
-  Og nå får du en blå bakgrunn istedenfor en grønn.
+  Hvilken farge får du nå?
 
 ## Forklaring av additive farger {.protip}
-
-Som nevnt har du sikkert lært om farger i skolen og blandet farger med
-maling, fargeblyanter eller -stifter. Da lærte du nok om primærfargene
-rødt, gult og blått, og at du kunne lage omtrent alle slags farger ved
-å blande disse sammen. Jo flere farger du blandet, desto mørkere ble
-den ferdige fargen. Dette er fordi du tegnet med fargede pigmenter som
-absorberer lys. Jo flere farger i lyset som absorberes, desto færre
-farger og mindre lys reflekteres og treffer øyet ditt.
-
-I en datamaskin er det annerledes. Du har nok lagt merke til at
-skjermen lyser. Derfor angir vi fargen som hvor mye lys vi skal lage
-av hver farge. Siden øyet reagerer på rødt, grønt og blått lys, er
-disse valgt som primærfargene i skjermer. Det er derfor de tre
-verdiene i kallet på `background` ga oss henholdsvis rødt, grønt og
-blått. Fordi primærfargene er rød, grønn og blå kalles dette systemet
-som regel RGB. Det er også kjent som et *additivt* system, fordi man
-legger sammen fargene, mens i tegning på papir jobber man med et
-*subtraktivt* system der farger trekkes fra.
-
-Vi kan også blande farger her, men det oppfører seg litt annerledes
-fra du kanskje er vant til fra maling. Hvis vi blander rødt og grønt,
+Som nevnt har du kanskje lært om farger på skolen og brukt maling eller
+fargeblyanter. Du lærte nok om primærfargene rød, gul og blå,
+og at du kunne lage omtrent alle slags farger ved å blande disse sammen.
+
+Når man maler, jo flere farger som blandes sammen, desto mørkere ble den

```

Comment:
Kanskje flytte leddsetningen "når man maler" til slutt i den første: Jo flere farger man blander sammen når man maler, desto mørkere blir den ferdige fargen. For å slippe så mange kommaer gjorde jeg den passive relativsetningen bare til en del av en aktiv setning.